internal/transactionsubmission/utils/utils.go

```diff
@@ -9,6 +9,8 @@ import (
 	sdpUtils "github.com/stellar/stellar-disbursement-platform-backend/internal/utils"
 )
 
+const AdvisoryLock = int(2172398390434160)
```

Comment:
💡  If this constant is only relevant to the channel accounts service, I believe it should live in the `channel_accounts_service.go` file.